src/main/java/it/gov/pagopa/payhub/auth/service/a2a/ValidateClientCredentialsService.java

```diff
@@ -0,0 +1,49 @@
+package it.gov.pagopa.payhub.auth.service.a2a;
+
+import it.gov.pagopa.payhub.auth.exception.custom.InvalidExchangeClientException;
+import it.gov.pagopa.payhub.auth.exception.custom.InvalidExchangeRequestException;
+import it.gov.pagopa.payhub.auth.exception.custom.InvalidGrantTypeException;
+import it.gov.pagopa.payhub.auth.model.Client;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.stereotype.Service;
+import org.springframework.util.StringUtils;
+
+@Service
+@Slf4j
+public class ValidateClientCredentialsService {
+	private final ClientService clientService;
+	public static final String ALLOWED_GRANT_TYPE = "client_credentials";
+	public static final String ALLOWED_SCOPE = "openid";
+
+	public ValidateClientCredentialsService(ClientService clientService) {
+		this.clientService = clientService;
+	}
+
+	public void validate(String clientId, String grantType, String scope, String clientSecret) {
+		validateClient(clientId);
```

Comment:
this class should just check that it has been provided! we should not check if it exists!
When we will authenticate the request with the next task, we will fetch it and verifying that the couple clientId/clientSecret exists